.changeset/hip-lamps-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/rag-ai-storage-pgvector': patch
```

Comment:
I believe this is a breaking change due to interface changes. Let's make it a `major` so we can start following proper semver

Reply:
updated!